### ISSUE TITLE
cannot import siibra==0.0.9.dev1

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#15]
environment:

```sh
> python --version
3.6.9
> pip --version
pip 21.0.1 from /home/xiao/dev/tmp/siibra-python/venv/lib/python3.6/site-packages/pip (python 3.6)
```

to reproduce:

```sh
> pip install siibra==0.0.9.dev1
> python
>>> import siibra as sb
```

error

```sh
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xiao/dev/tmp/siibra-python/venv/lib/python3.6/site-packages/siibra/__init__.py", line 26, in <module>
    with open(path.join(PKG_DIR,"..","VERSION"),"r") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/xiao/dev/tmp/siibra-python/venv/lib/python3.6/site-packages/siibra/../VERSION'

```

**Comment by dickscheid** [comment]
I guess the problem was the location of the VERSION file in thte root folder, it should be located in the package folder siibra/.

**Comment by dickscheid** [comment]
This should be  fixed by merging your PR. New pypi package for updated version 0.0.9.dev2 has been created.

**Comment by xgui3783** [comment]
incidentally, are you publishing the repository manually?

we can perhaps try to automate that in the future. (then tests can also be triggered automatically)

I can confirm that this issue is resolved in 0.0.9.dev2
